shared/fixes/bash/file_groupowner_etc_shadow.sh

```diff
@@ -0,0 +1,2 @@
+# platform = multi_platform_all
+chgrp root /etc/shadow
```

Comment:
Not required, shared/templates/create_file_groupowner.py should generate this fix for you from template